potimeb/An_Unholy_Rock_Fiend.lua

```diff
@@ -1,4 +1,17 @@
+--An_Unholy_Rock_Fiend (223125)
+--Phase 2 
+--potimeb
+
 function event_death_complete(e)
-	-- send a signal to the zone_status that I died
-	eq.signal(223097,3); -- NPC: zone_status
+	--signal phase_two_controller
+	eq.signal(223242,1);
+end
+
+function event_signal(e)
+	--signal from A Windshapen Warlord
+	if e.signal == 1 then
+		e.self:SetSpecialAbility(25, 0);
+		e.self:SetSpecialAbility(24, 0);
+		e.self:SetBodyType(1, true);
```

Comment:
Should use `SpecialAbility.immune_aggro`, `SpecialAbility.immune_aggro_on` and `BT.Humanoid` here instead (I know we don't do this perfectly everywhere, but we should!)